mcs_benchmark_data/pipelines/commonsense_qa/commonsense_qa_small_benchmark_pipeline.py

```diff
@@ -0,0 +1,25 @@
+from mcs_benchmark_data._pipeline import _Pipeline
+
+from mcs_benchmark_data.pipelines.commonsense_qa.commonsense_qa_small_benchmark_extractor import (
+    CommonsenseQaSmallBenchmarkExtractor,
+)
+from mcs_benchmark_data.pipelines.commonsense_qa.commonsense_qa_benchmark_transformer import (
+    CommonsenseQaBenchmarkTransformer,
+)
+
+
+class CommonsenseQaSmallBenchmarkPipeline(_Pipeline):
```

Comment:
Instead of creating all of this parallel code for the small data sets, please parameterize the pipeline and the extractor as I suggested. Then you're testing the real code, just with a smaller dataset. With this setup you're not testing the real code.

Reply:
In general you should strive to minimize changes between "testing mode" and "production mode". It's this principle Raymond Chen captured well:
https://devblogs.microsoft.com/oldnewthing/20060815-00/?p=30113